projects/shared/src/lib/components/floor/floor.component.scss

```diff
@@ -0,0 +1,136 @@
+@import "../../styles-lib/colors.scss";
+
+.floor-number {
+  display: block;
+  margin-bottom: 10px;
+  text-align: center;
+  font-size: 5rem;
+}
+
+.back-button {
+  display: block;
+  min-height: 40px;
+  border: 2px solid $SecondColor;
```

Comment:
Change to $SecondaryColor or choose another name. You can use some service like http://chir.ag/projects/name-that-color/